src/modules/credentials/CredentialsModule.ts

```diff
@@ -85,6 +84,8 @@ export class CredentialsModule {
 
     const credentialDefinitionId = config?.credentialDefinitionId ?? credentialProposalMessage.credentialDefinitionId
 
+    credentialRecord.attachments = credentialProposalMessage.attachments
```

Comment:
I think we only want to save the linked attachments

```suggestion
    credentialRecord.linkedAttachments = credentialProposalMessage.attachments.filter(....)
```

Reply:
The attachments in the `credentialProposalMessage` are just regular `attachments` and not `linkedAttachments`. I think storing them as regular attachments is a better solution as it can also help with other use cases.